manuscript/13-SoftSec.txt

```diff
@@ -1,18 +1,141 @@
-{mainmatter}
+# Ασφάλεια λογισμικού {#softsec}
 
-# TODO {#TODO}
 
 ## Ακρωνύμια
 
-|ΑΚΡΩΝΥΜΙΟ |Επεξήγηση   |
+|CC  |Common Criteria              |
+|SDLC|Secure development life cycle|
 
 ## Εισαγωγή
 
+Η ασφάλεια έχει εξελιχθεί σε καθοριστικό παράγοντα στην ανάπτυξη λογισμικού εξαιτίας της διαρκώς αυξανόμενης κακόβουλης δραστηριότητας, 
+η οποία στοχοποιεί τα υπολογιστικά συστήματα. Επιπλέον των συνήθων ζητημάτων  ορθότητας και αξιοπιστίας, οι προγραμματιστές
+οφείλουν να επιδεικνύουν ιδιαίτερη προσοχή στην ασφάλεια του λογισμικού το οποίο αναπτύσσουν. 
+Η ασφαλής διαδικασία ανάπτυξης λογισμικού επιτυγχάνει την ασφάλεια μέσω της εφαρμογής ενός συνόλου καθιερωμένων ή/και προτεινόμενων κανόνων και πρακτικών.
+Στο ίδιο πλαίσιο, η ασφαλής διαδικασία συντήρησης λογισμικού συμπληρώνει την ασφαλή ανάπτυξη, εξασφαλίζοντας την αποφυγή εισαγωγής ζητημάτων ασφαλείας 
+κατά την διάρκεια της διαδικασίας συντήρησης.
 
-## Κατανομή των θεμάτων για TODO
+## Κατανομή των θεμάτων για την ασφάλεια λογισμικού
 
```

Comment:
do not forget to add this line

The breakdown of topics for the Software Security knowledge area (KA) is shown in Figure 13.1.